### PR TITLE
[FIX] Fixing bug in jumpToOrphans (master).

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

File: include/seqan/stream/formatted_file.h [hunk 1]
```diff
@@ -400,6 +400,11 @@ struct FormattedFileContext
     typedef Nothing Type;
 };
 
+template <typename TFormattedFile, typename TStorageSpec>
+struct FormattedFileContext<TFormattedFile const, TStorageSpec> :
```
Comment (esiragusa) [inline include/seqan/stream/formatted_file.h]: If the file is const, its context should also be const, right?

Comment (holtgrewe) [inline include/seqan/stream/formatted_file.h]: Hm, `TStorageSpec` is something like `Dependent<>` or `Owner<>`. I am not sure about the implications of making tags consts, we used to have problems with const vs. non-const tags. What do you think?

File: include/seqan/stream/formatted_file.h [hunk 2]
```diff
@@ -400,6 +400,11 @@ struct FormattedFileContext
...
+    FormattedFileContext<TFormattedFile, TStorageSpec>
```
Comment (esiragusa) [inline include/seqan/stream/formatted_file.h]: Sorry, I should have commented the line below... I'm not referring to TStorageSpec. The FormattedFileContext of a TFormattedFile const should be a FormattedFileContext::Type const.

Comment (esiragusa) [inline include/seqan/stream/formatted_file.h]: Hey @holtgrewe isn't this what you want?

```cpp
template <typename TFormattedFile, typename TStorageSpec>
struct FormattedFileContext<TFormattedFile const, TStorageSpec>
{
    typedef FormattedFileContext<TFormattedFile, TStorageSpec> const Type;
};
```

**Comment by esiragusa** [comment]
@holtgrewe could you please answer my question about the FormattedFileContext? So I can quickly merge the PR. Thanks!

File: include/seqan/stream/formatted_file.h (hunk 2 above)
Comment (holtgrewe) [inline include/seqan/stream/formatted_file.h]: I guess that would work.

Comment (esiragusa) [inline include/seqan/stream/formatted_file.h]: Can you please change that in your PR? I cannot...

**Comment by holtgrewe** [comment]
@esiragusa OK, hope that's fine now.

**Pull request merged**